ws-server/src/server.rs

```diff
@@ -56,7 +56,7 @@ const MAX_CONNECTIONS: u64 = 100;
 
 /// A WebSocket JSON RPC server.
 #[derive(Debug)]
-pub struct Server<M = ()> {
+pub struct Server<M> {
```

Comment:
Why remove the default?

Reply:
Was trying to make `Builder::default()` work and this seemed to be in the way, but I don't think this was the problem. OTOH: do we need it?

Reply:
I'd keep it, all it does is if you don't use middleware you can just write `WsServer` instead of `WsServer<()>`.